src/commands/TimeCommand.ts

```diff
@@ -0,0 +1,157 @@
+import { Command } from "../Command";
+import { CommandInteraction, GuildMember } from "discord.js";
+import {
+	SlashCommandBooleanOption,
+	SlashCommandBuilder,
+	SlashCommandIntegerOption,
+	SlashCommandStringOption,
+	SlashCommandSubcommandsOnlyBuilder,
+} from "@discordjs/builders";
+
+class Reminder extends Command {
+	constructor() {
+		super("reminder");
+	}
+
+	elapse(interaction: CommandInteraction, callAll: boolean, caller: string, message = ""): void {
```

Comment:
passing a boolean into a function is bad practice. maybe just pass the ping string instead.

Reply:
why is putting boolean into a function bad?